### PR TITLE
ISPN-6913: Cache loader undo type change always returns to 'Custom St…

### PR DESCRIPTION
**Pull request by ryanemerson**
…ore'

https://issues.jboss.org/browse/ISPN-6913

### CODE REVIEW

File: src/main/webapp/components/directives/cache/loaders/cache-loader.js [hunk 1]
```diff
@@ -178,8 +174,9 @@
         scope.fieldValueModified = function (field) {
           var original = scope.prevData[field];
           var latest = scope.data[field];
+          var customLoaderSelected = field === 'class' && latest === '' && utils.isNullOrUndefined(original);
 
-          if (((utils.isNullOrUndefined(original) || original === '') && !latest) || original === latest) {
+          if (!customLoaderSelected && ((utils.isNullOrUndefined(original) || original === '') && !latest) || original === latest) {
```
Comment (vblagoje) [inline src/main/webapp/components/directives/cache/loaders/cache-loader.js]: @ryanemerson does this logic really have to be this complicated? I can hardly follow it :-(

Comment (ryanemerson) [inline src/main/webapp/components/directives/cache/loaders/cache-loader.js]: @vblagoje Good point, I'll have another look and try to simplify it and/or break it down into multiple functions.

**Comment by ryanemerson** [comment]
@vblagoje Update to try and improve clarity

**Comment by ryanemerson** [comment]
Closing as [ISPN-7081](https://issues.jboss.org/browse/ISPN-7081) makes this redundant.

**Pull request closed without merging**